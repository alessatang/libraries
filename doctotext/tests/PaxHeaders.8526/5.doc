28 atime=1389151473.2078872
28 ctime=1389151473.2078872
