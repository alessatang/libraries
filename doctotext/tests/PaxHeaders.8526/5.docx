30 atime=1389151473.214887172
30 ctime=1389151473.214887172
