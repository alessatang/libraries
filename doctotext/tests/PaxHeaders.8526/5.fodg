30 mtime=1384978086.853497475
30 atime=1389151473.214887172
30 ctime=1389151473.214887172
