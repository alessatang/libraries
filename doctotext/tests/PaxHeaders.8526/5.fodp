30 mtime=1384978086.856497476
30 atime=1389151473.345886635
30 ctime=1389151473.345886635
