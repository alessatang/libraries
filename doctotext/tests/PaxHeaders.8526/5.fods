30 mtime=1384978086.857497477
30 atime=1389151473.345886635
30 ctime=1389151473.345886635
