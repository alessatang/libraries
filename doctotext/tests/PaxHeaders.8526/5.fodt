30 mtime=1384978086.858497477
30 atime=1389151473.345886635
30 ctime=1389151473.345886635
