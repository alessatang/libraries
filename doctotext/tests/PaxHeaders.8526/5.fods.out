30 mtime=1384978086.901497491
30 atime=1389151473.345886635
30 ctime=1389151473.345886635
